nixos/modules/services/web-apps/tt-rss.nix

```diff
@@ -562,14 +575,14 @@ let
               if cfg.database.type == "pgsql" then ''
                   ${optionalString (cfg.database.password != null) "PGPASSWORD=${cfg.database.password}"} \
                   ${optionalString (cfg.database.passwordFile != null) "PGPASSWORD=$(cat ${cfg.database.passwordFile})"} \
-                  ${pkgs.sudo}/bin/sudo -u ${cfg.user} ${config.services.postgresql.package}/bin/psql \
+                  ${config.services.postgresql.package}/bin/psql \
                     -U ${cfg.database.user} \
                     ${optionalString (cfg.database.host != null) "-h ${cfg.database.host} --port ${toString dbPort}"} \
                     -c '${e}' \
                     ${cfg.database.name}''
 
               else if cfg.database.type == "mysql" then ''
-                  echo '${e}' | ${pkgs.sudo}/bin/sudo -u ${cfg.user} ${config.services.mysql.package}/bin/mysql \
+                  echo '${e}' | ${config.services.mysql.package}/bin/mysql \
```

Comment:
can we extract the "pipe to mysql/postgresql" command line generation into a separate function? This will simplify moving it to a separate file and reusing from there, in case other modules need it too…

Reply:
@flokli I'm not sure what you mean by this, as it is already declared as a function:
https://github.com/NixOS/nixpkgs/blob/1cd3b98c3a7e93c95190e4d3fb76cca68cb9e049/nixos/modules/services/web-apps/tt-rss.nix#L574
Can you please clarify?

Reply:
I meant having this more generic, so that the logic to assemble the cmdline can be used from other modules as well (like the roundcube module).

Currently, this function still uses values from `cfg(.database)`, for which we don't yet have a defined interface - we could move that function into https://github.com/NixOS/nixpkgs/issues/62073 though…

Nothing necessary to do for this PR, but probably still something good to keep in mind.

Reply:
Ah yes I see what you're saying. I hadn't thought of that. Good idea. I should really get around to #62073 at some point...